tests/sidetrail/working/s2n-record-read-cbc-negative-test/stubs/s2n_cbc.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <smack.h>
+#include "ct-verif.h"
+#include "../sidetrail.h"
+
+#define MAX_LEAKAGE_DIFFERENCE  68
```

Comment:
Realizing this matches the positive test, but a comment on where this came from would be helpful (the README talks about the what, but not where `68` came from).

Reply:
This was based off the MAX_LEAKAGE for the cbc module, verified here: https://github.com/awslabs/s2n/blob/master/tests/sidetrail/working/s2n-cbc/cbc.c#L38